src/watch-actions/action-renamed.js

```diff
@@ -20,7 +18,8 @@ class ActionRenamed {
     init() {
         this.assetId = this.conf.store.getAssetId(this.data.remoteOldPath);
 
-        this.parentId = this.conf.store.getAssetId(this.data.remoteNewDir);
+        this.parentId = this.data.remoteNewDir !== '.' &&
```

Comment:
Here, the type of `parentId` is **(boolean | number)**. It is fine since `parent: this.parentId || NO_PARENT_TOKEN` and _parent_ is **(null | number)**.

Is it possible `getAssetId(asset) === 0`?(`this.parentId === 0`)

Reply:
parentRemote in ActionCreated is always an empty string ('') for the root dir. I changed remoteNewDir to also be '' instead of '.', for consistency. It's better to handle the case when asset id is zero, even though that's impossible in practice.